libsrc/Makefile

```diff
@@ -91,7 +93,7 @@ define INSTALL_recipe
 
 $(if $(prefix),,$(error variable `prefix' must be set))
```

Comment:
> [...] isn't that kind of broken here?

No, it's as it is intended to be.

Following standards (https://www.gnu.org/prep/standards/html_node/DESTDIR.html) $DESTDIR is optional.

However in constrast to standards (https://www.gnu.org/prep/standards/html_node/Directory-Variables.html) $prefix isn't optional, therefore the explicit check.